### PR TITLE
Address unchanged files with check annotations

### PR DESCRIPTION
**Pull request by AdrianaCeric**
**Description:**
These are annoying:
Pasted Graphic
I eslinted these files so they don't show up in every PR review

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Comment by AdrianaCeric** [comment]
oh I realized that there are more unchecked files with type annotations ....... 😮

**Comment by miles-grant-ibigroup** [comment]
There's hundreds!

**Comment by binh-dam-ibigroup** [comment]
@AdrianaCeric Could you fix the failed test from `test-build-release` from the checks? I looks like `yarn unit` failed somehow.

File: __tests__/test-utils/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // Sun Aug 04 2019 19:34:56 GMT-0700
 const DEFAULT_TEST_TIME = Date.UTC(2019, 7, 5, 2, 34, 56, 78)
 
-export function timeoutPromise (ms) {
-  return new Promise((resolve, reject) => {
+export function timeoutPromise(ms: number): Promise<void> {
```
Comment (binh-dam-ibigroup) [inline __tests__/test-utils/index.js:4]: Now that you have added Typescript types, rename this file to `index.ts`. With that I think the failing `yarn unit` step in the checks should pass.

File: __tests__/util/itinerary.js
Comment (binh-dam-ibigroup) [inline __tests__/util/itinerary.js:1]: Also rename this file to `itinerary.ts`.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Rename files to use `.ts` extension, that should fix the failing test.

File: __tests__/test-utils/index.js (hunk 1 above)
Comment (AdrianaCeric) [reply __tests__/test-utils/index.js:4]: Ok, thank you! Was wondering what was causing those

**Review by miles-grant-ibigroup — APPROVED** [review]
Working great! Just a small nitpick

**Comment by AdrianaCeric** [comment]
Kind of debating whether I want to fix the other unchecked files in this pr. There are 35 more files

**Pull request merged**